STYLEGUIDE.md

```diff
@@ -8,70 +8,79 @@ the version on the master branch should be followed, as this version should be t
+5. [Formatting](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#formatting)
+   1. [Braces](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#braces) 
+      1. [One true brace style (1TBS)](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#one-true-brace-style-1TBS) 
+   2. [100 character column limit ](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#100-character-column-limit)
+   3. [One statement per line](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#one-statement-per-line)
+   4. [Empty blocks can be concise](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#empty-blocks-can-be-concise)
+   5. [Whitespace](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#whitespace)
       1. [Vertical whitespace](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#vertical-whitespace)
-      1. [No horizontal alignment](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#no-horizontal-alignment)
-   1. [Optional grouping parentheses: recommended](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#optional-grouping-parentheses-recommended)
-   1. [Enum classes](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md)
-      1. [Comment fall-through](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md)
-      1. [Enumerate all cases or have a default case](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enumerate-all-cases-or-have-a-default-case)
-   1. [Annotations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#annotations)
-   1. [Comments](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comments)
-   1. [Modifiers](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#modifiers)
-   1. [Numeric literals](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#numeric-literals)
+      2. [No horizontal alignment](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#no-horizontal-alignment)
+   6. [Optional grouping parentheses: recommended](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#optional-grouping-parentheses-recommended)
+   7. [Enum classes](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enum-classes)
+      1. [Comment fall-through](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comment-fall-through)
+      2. [Enumerate all cases or have a default case](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#enumerate-all-cases-or-have-a-default-case)
+   8. [Annotations](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#annotations)
+   9.  [Comments](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#comments)
+      1.  [Do you need the comment?](https://github.com/knjk04/book-project/blob/master/STYLEGUIDE.md#do-you-need-the-comment)
```

Comment:
There's an extra space before the opening square bracket. I think this is what is preventing this section from being indented at the right level in the table of contents